### PR TITLE
[Replace Non-Blocking Usage] LPS-198502 Replace non-blocking ServiceProxyFactory usages in classes ServiceAccessPolicyManagerUtil and TunnelAuthenticationManagerUtil

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:startup

**Comment by W2jx** [comment]
ci:test:upgrade

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by catarineaguiar** [comment]
ci:test:upgrade

**Comment by W2jx** [comment]
no conflicts(10/30 7:57)

**Comment by W2jx** [comment]
no conflicts(10/30 7:57)

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by Kvale1733** [comment]
ci:test:upgrade

**Comment by W2jx** [comment]
no conflicts(11/1 9:03)

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by W2jx** [comment]
no conflicts(11/2 8:14)

**Comment by W2jx** [comment]
no conflicts(11/3 9:19)

File: portal-impl/src/META-INF/security-spring.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
 	<bean class="com.liferay.portal.kernel.security.auth.DefaultFullNameValidator" id="com.liferay.portal.kernel.security.auth.DefaultFullNameValidator" />
 	<bean class="com.liferay.portal.kernel.security.auth.DefaultScreenNameGenerator" id="com.liferay.portal.kernel.security.auth.DefaultScreenNameGenerator" />
 	<bean class="com.liferay.portal.kernel.security.auth.DefaultScreenNameValidator" id="com.liferay.portal.kernel.security.auth.DefaultScreenNameValidator" />
+	<bean class="com.liferay.portal.security.auth.tunnel.TunnelAuthenticationManagerImpl" id="com.liferay.portal.kernel.security.auth.tunnel.TunnelAuthenticationManager" />
```
Comment (dantewang) [inline portal-impl/src/META-INF/security-spring.xml:23]: @W2jx There's already a `TunnelAuthenticationManagerImpl` bean in security-spring.xml.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
Plz resend, thanks.

**Pull request closed without merging**